build.gradle

```diff
@@ -85,6 +85,21 @@ ext {
     junit_version = "5.8.2"
 }
 
+allprojects {
+    apply plugin: "com.diffplug.spotless"
+    spotless {
+        java {
```

Comment:
I think that we could also add the `ratchetFrom 'origin/main'` feature. If I understand the ratchet feature correctly, this make Spotless apply to only files that differ from `origin/main`. This could make our transition easier as we would only need fix the formatting of files as we work on them instead of pushing a massive commit to **all** the files that are not formatted correctly.

https://github.com/diffplug/spotless/tree/main/plugin-gradle#ratchet